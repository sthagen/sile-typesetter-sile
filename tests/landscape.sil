\begin[landscape=true]{document}
\end{document}
