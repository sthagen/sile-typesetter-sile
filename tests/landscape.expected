Set paper size 	841.8897729	595.275597
Begin page
Mx 	418.5987
My 	553.7327
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	20 w=4.6924	(1)
End page
Finish
